### DISCUSSION TITLE
Windows silent install

### DISCUSSION THREAD

**Discussion by lishniy** [transmission/transmission#2684] [Q&A]
Where i can find parameters for silent isntall from msi package?

**Comment by mikedld** [comment]
Those aren't documented since there was no need in customizing anything. Are there any particular parameters you're looking for? What's your use-case?

**Comment by lishniy** [comment]
I want build windows container image. Looking for parameters to install daemon

**Comment by mikedld** [comment]
There's a number of features available for selection during installation: https://github.com/transmission/transmission/blob/54a7cac25ad4f8d47a3198e4b2eeec13baeea99d/dist/msi/Transmission.wxs#L55-L74

Each feature has an `Id` which (according to MSDN) could be used with (among others) [`ADDLOCAL`](https://docs.microsoft.com/en-us/windows/win32/msi/addlocal) and [`REMOVE`](https://docs.microsoft.com/en-us/windows/win32/msi/remove) properties. Given that Qt client is selected and all the rest are deselected by default, you could (I didn't myself) try e.g.

```
msiexec /quiet /i transmission.msi ADDLOCAL=Daemon REMOVE=QtClient
```
